app/errors.js

```diff
@@ -1,7 +1,16 @@
+var log = require('bole')('error-handler');
+var sentry = require('raven');
 var url_for = require('./routes').url_for;
 
 
+process.on('unhandledRejection', function (error) {
```

Comment:
What is this `unhandledRejection` event?

Reply:
Oh, [it's happening when a promise's error is not handled](https://nodejs.org/api/process.html#process_event_unhandledrejection).